test/service.go

```diff
@@ -57,8 +77,8 @@ func WithServiceReady(ctx *Context, name, namespace, image string) (*servingv1.S
 	return service, nil
 }
 
-func CreateService(ctx *Context, name, namespace, image string) (*servingv1.Service, error) {
-	service, err := ctx.Clients.Serving.ServingV1().Services(namespace).Create(context.Background(), Service(name, namespace, image, nil), metav1.CreateOptions{})
+func CreateService(ctx *Context, name, namespace, image string, annotations map[string]string) (*servingv1.Service, error) {
```

Comment:
```suggestion
func CreateService(ctx *Context, name, namespace, image string) (*servingv1.Service, error) {
```

I think we don't need this change for now as it always passes `nil`.